<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Qlabel_weclome">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Welcome to Cat Chaser!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>281</width>
      <height>474</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout_left">
     <item>
      <widget class="QRadioButton" name="button_charlie">
       <property name="text">
        <string>Charlie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="charliepic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/charlie.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_grumpy">
       <property name="text">
        <string>Grumpy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="grumpypic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/grumpy</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_nyancat">
       <property name="text">
        <string>Nyan Cat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nayncatpic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/nyan2.jpg</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>241</width>
      <height>497</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout_right">
     <item>
      <widget class="QRadioButton" name="buttonmouse">
       <property name="text">
        <string>Jerry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mousepic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/mouse.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttondog">
       <property name="text">
        <string>Edgar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dogpic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/dog.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonhuman">
       <property name="text">
        <string>Dee Dee</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="humanpic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/image/Deedee.gif</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Qlabe_choosecat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Who are you?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Qlabel_chooseenemy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Who's your chaser?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startbutton">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>561</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instruction_button">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>521</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>INSTRUCTION</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
